_posts/2023-03-31-announcing-trino-fest-2023.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "Announcing Trino Fest 2023"
+author: "Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/trino-fest.png
+---
+
+Trino Fest 2023: Lakehouse Summer Camp
+
+Get ready to kick off your summer with Commander Bun Bun at Trino Fest 2023!
+This year's event is going virtual and will take place over two days, June 14th
+and June 15th. The focus of the event will be on Trino as a data lakehouse query
+engine, with discussions on new features and the ecosystem around Trino to
+support better data lakehouse management to keep up with data warehouses.
+
+Trino Fest 2023 is the new annual Summer event dedicated to all things Trino.
+Building on the success of last year’s [Cinco de Trino]
+({% post_url 2022-05-17-cinco-de-trino-recap %}), we’re excited to bring the
+community together once again to explore the latest trends and innovations in
+Trino and data lakehouse management. With a focus on educating, community
+collaboration, and inspiration, Trino Fest 2023 will be a valuable experience
+for anyone interested in lowering their analytics costs while still maintaining
+the features from a data warehouse. We hope to see you there!
+
+<!--more-->
+
+[Call for papers (CFP)](https://sessionize.com/trino-fest-2023) is now open,
+and we invite you to submit a talk if you have an interesting perspective on
+Trino. We're particularly interested in talks related to:
+
+* data lake use cases, such as data lake architecture
+* Apache Iceberg
+* Delta Lake
+* Hudi
+* industry use cases for Trino
+* query federation
+* data governance with Trino
+* SQL in Trino that enables data warehouse features on the data lake
+* ETL/ELT/batch query processing using fault tolerance-execution
+* tools in the Trino and data lake ecosystem
+
+The CFP closes on April 28th, so be sure to submit your talk soon!
+
+Aside from the new title, this year's Trino Fest will differ from last years,
+short conference in a few ways. We're featuring more talks from Trino
+practitioners, the event will be divided into two days to avoid the death march
+of talks, and there will be more Summer, lakehouse, and camping puns. Of course,
+there will be continued use of the Trinoritaville song. Whether you're just
+getting started with Trino or you're a seasoned pro, there will be something for
+everyone at Trino Fest.
+
+Trino was [built from the beginning to run on Hive]({% post_url
+2020-10-20-intro-to-hive-connector %}), so Trino running on a data lake is
```

Comment:
what do you mean ... Hive usage is a data lake..

Reply:
I mean Hive is like the OG data lake. It was one of the early big data query engine abstractions that build on a separate storage system. Data lake to me is just distributed storage that takes any variety and data and implies a schema over it to make it queryable. That is what Hive was and what Trino is today.